lib/python/ensembl/brc4/runnable/process_seq_region.py

```diff
@@ -382,6 +421,39 @@ def get_codon_table(self, record: SeqRecord) -> int:
                 break
         
         return table
+    
+    def get_organelle(self, record: SeqRecord) -> str:
+        """Given a genbank record, look for an organelle field.
+
+        Args:
+            record: The GenBank record to look into.
+
+        Returns:
+            the organelle location (empty string if not found).
+        """
```

Comment:
A shorter docstring could be:
```suggestion
        """Returns the organelle location from the given GenBank record, or an empty string if not found.

        Args:
            record: The GenBank record to look into.

        """
```